docs/features/job-level-affinity.md

```diff
@@ -0,0 +1,148 @@
+godel-demo-worker5         Ready    <none>                 36m   v1.21.1   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/arch=amd64,kubernetes.io/hostname=godel-demo-worker5,kubernetes.io/os=linux,netipv4=10.53.0.0,netminiipv4=10.53.16.0,subCluster=subCluster-a
+godel-demo-worker6         Ready    <none>                 36m   v1.21.1   beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,kubernetes.io/arch=amd64,kubernetes.io/hostname=godel-demo-worker6,kubernetes.io/os=linux,netipv4=10.57.0.0,netminiipv4=10.57.111.0,subCluster=subCluster-b
+```
+- **godel-demo-worker** and **godel-demo-worker4** share the same 'netminiipv4';
+- **godel-demo-worker**, **godel-demo-worker-3**, and **godel-demo-worker4** share the same 'netipv4'.
+
+Second, create the podgroup and deployment.
+```
+$ kubectl apply -f manifests/quickstart-feature-examples/job-level-affinity/podGroup.yaml
+podgroup.scheduling.godel.kubewharf.io/nginx created
+
+$ kubectl apply -f manifests/quickstart-feature-examples/job-level-affinity/deployment.yaml
+deployment.apps/nginx created
+```
+
+Third, check the scheduling result.
+```
+$ kubectl get pods -l name=nginx -o wide
+NAME                     READY   STATUS    RESTARTS   AGE   IP            NODE                 NOMINATED NODE   READINESS GATES
+nginx-68fc9649cc-5pdb7   1/1     Running   0          8s    10.244.2.21   godel-demo-worker    <none>           <none>
+nginx-68fc9649cc-b26tk   1/1     Running   0          8s    10.244.2.19   godel-demo-worker    <none>           <none>
+nginx-68fc9649cc-bvvx6   1/1     Running   0          8s    10.244.2.18   godel-demo-worker    <none>           <none>
+nginx-68fc9649cc-dtxqn   1/1     Running   0          8s    10.244.2.23   godel-demo-worker    <none>           <none>
+nginx-68fc9649cc-hh5pr   1/1     Running   0          8s    10.244.3.34   godel-demo-worker4   <none>           <none>
+nginx-68fc9649cc-jt8q9   1/1     Running   0          8s    10.244.3.35   godel-demo-worker4   <none>           <none>
+nginx-68fc9649cc-l8j2s   1/1     Running   0          8s    10.244.2.20   godel-demo-worker    <none>           <none>
+nginx-68fc9649cc-t9fb8   1/1     Running   0          8s    10.244.3.33   godel-demo-worker4   <none>           <none>
+nginx-68fc9649cc-vcjm7   1/1     Running   0          8s    10.244.2.17   godel-demo-worker    <none>           <none>
+nginx-68fc9649cc-wplt7   1/1     Running   0          8s    10.244.2.22   godel-demo-worker    <none>           <none>
+```
+The pods have been scheduled to **godel-demo-worker** and **godel-demo-worker4**, which share the same 'netminiipv4'. We achieved this result because the resources are sufficient.
+
+Next, let's try increasing the pod count to 15.
```

Comment:
ATTENTION: The 'Spec' field of PodGroup should not be modified.

All previously Pods and PodGroup should be cleaned up, and the scheduler components should be restarted to avoid the legacy impact of podgroup with the same name.

Reply:
lgtm otherwise

Reply:
I see. Added podGroup**-2**.yaml and deployment**-2**.yaml for the 15 replica, instead of updating the original one.